<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_ActuatorElectrobot</class>
 <widget class="QMainWindow" name="ihm_ActuatorElectrobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Moteurs">
       <attribute name="title">
        <string>Moteurs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_Moteur_1">
          <property name="text">
           <string>Moteur 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="Moteur_1">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Moteur_2">
          <property name="text">
           <string>Moteur 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="Moteur_2">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_Moteur_3">
          <property name="text">
           <string>Moteur 3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="Moteur_3">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2" rowspan="2">
         <widget class="QPushButton" name="Moteurs_StopAll">
          <property name="text">
           <string>RAZ ALL</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_Moteur_4">
          <property name="text">
           <string>Moteur 4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="Moteur_4">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_Moteur_5">
          <property name="text">
           <string>Moteur 5</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="Moteur_5">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_Moteur_6">
          <property name="text">
           <string>Moteur 6</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="Moteur_6">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="Send_CdeMoteurs">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;clic gauche : envoie trame / clic droit : lock trame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Send /  Lock</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Servos">
       <attribute name="title">
        <string>Servos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="servo_label_1">
          <property name="text">
           <string>Servo20</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="Servo_slide_20">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="Servo_num_20">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="servo_label_2">
          <property name="text">
           <string>Servo19</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="Servo_slide_19">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="Servo_num_19">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="servo_label_3">
          <property name="text">
           <string>Servo18</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="Servo_slide_18">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="Servo_num_18">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="servo_label_4">
          <property name="text">
           <string>Servo17</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="Servo_slide_17">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="Servo_num_17">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="servo_label_5">
          <property name="text">
           <string>Servo16</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="Servo_slide_16">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="Servo_num_16">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="servo_label_6">
          <property name="text">
           <string>Servo15</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="Servo_slide_15">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="Servo_num_15">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="servo_label_7">
          <property name="text">
           <string>Servo14</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="Servo_slide_14">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSpinBox" name="Servo_num_14">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="servo_label_8">
          <property name="text">
           <string>Servo13</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSlider" name="Servo_slide_13">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QSpinBox" name="Servo_num_13">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>31</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="1" colspan="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>226</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ServosAX">
       <attribute name="title">
        <string>ServosAX</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="1">
         <widget class="QSlider" name="ServoAX_slide_4">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QSpinBox" name="ServoAX_num_7">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="servoAX_label_2">
          <property name="text">
           <string>ServoAX 1</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="ServoAX_slide_5">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="servoAX_label_7">
          <property name="text">
           <string>ServoAX 6</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>62</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="ServoAX_num_0">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="ServoAX_slide_3">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="ServoAX_slide_0">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="servoAX_label_5">
          <property name="text">
           <string>ServoAX 4</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="ServoAX_slide_6">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="ServoAX_num_3">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="ServoAX_num_2">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="ServoAX_num_1">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="servoAX_label_8">
          <property name="text">
           <string>ServoAX 7</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="servoAX_label_3">
          <property name="text">
           <string>ServoAX 2</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="ServoAX_num_5">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="servoAX_label_4">
          <property name="text">
           <string>ServoAX 3</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="ServoAX_num_4">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="servoAX_label_6">
          <property name="text">
           <string>ServoAX 5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="ServoAX_slide_2">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="ServoAX_slide_1">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSlider" name="ServoAX_slide_7">
          <property name="maximum">
           <number>1023</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSpinBox" name="ServoAX_num_6">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="servoAX_label_1">
          <property name="text">
           <string>ServoAX 0</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>226</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ConfigServosAX">
       <attribute name="title">
        <string>Config ServosAX</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="3" column="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>168</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>ID servo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="ServosAXConfig_ID">
          <property name="toolTip">
           <string>ID [0 ; 253]
254 = tous les servos sont concernés</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Action</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QComboBox" name="ServosAXConfig_Action"/>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="ServosAXConfig_Value">
          <property name="maximum">
           <number>1023</number>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QPushButton" name="ServosAXConfig_Send">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
